### ISSUE TITLE
Save game progress

### ISSUE THREAD

**Issue by mscroggs** [AVEgame/AVE#121]
Allow the player to save and load their current game position.

**Comment by Giannie** [comment]
I feel like this should be easy now that progress is all stored in the character.

Maybe game loading could be added to the game title screen to avoid needing to also store the game ID.

Game title screen could have:
* Start a new game
* Load progress from file
* Play a different game

Data could simply be stored in JSON. This would also mean that we could do easy debugging of games since we’ll be able to get to the same game state as a user.

We could also automatically save game state on an error.

**Comment by Giannie** [comment]
We could add a curses shortcut to do this to save screen real estate.

**Comment by mscroggs** [comment]
Yes, I think this is now very easy, but I'm holding off until after version 2.0
